spec/lib/miq_automation_engine/models/miq_ae_git_spec.rb

```diff
@@ -0,0 +1,157 @@
+require 'spec_helper'
+
+describe MiqAeGit do
+  context "repository" do
+    before do
+      git_db    = "TestGit.git"
+      @ae_db_dir = Dir.mktmpdir
+      @default_hash = {:a => "one", :b => "two", :c => "three"}
+      @dirnames  = %w(A B c)
+      @repo_path   = File.join(@ae_db_dir, git_db)
+      @filenames = %w(A/File1.YamL B/File2.YamL c/File3.YAML)
+      @deleted_names = %w(A A/File1.YamL)
+      @conflict_file = 'A/File1.YamL'
+      @master_url = "file://#{@repo_path}"
+      options = {:path  => @repo_path,
+                 :name  => "user1",
+                 :email => "user1@example.com",
+                 :bare  => true,
+                 :new   => true}
+      @ae_db = MiqAeGit.new(options)
+      @original_commit = add_files_to_bare_repo(@filenames)
+    end
+
+    after do
+      FileUtils.remove_entry_secure(@ae_db_dir) if Dir.exist?(@ae_db_dir)
```

Comment:
Can we just use a normal `FileUtils.rm_f`...don't understand the need for remove_entry_secure